extensions/wikia/AdEngine/js/spec/lookup/adapter/appnexus.spec.js

```diff
@@ -14,6 +15,11 @@ describe('ext.wikia.adEngine.lookup.adapter.appnexus', function () {
 		},
 		geo: {
 			isProperGeo: jasmine.createSpy('isProperGeo')
+		},
+		appNexusPlacements: {
+			getPlacement: function() {
+				return '123'
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ASemicolon)